### ISSUE TITLE
Color coding and accessibility

### ISSUE THREAD

**Issue by stanleykc** [UnityFoundation-io/Libre311#282]
Color coding of service requests is not clear. The meaning of the colors is not defined, and is also an issue with accessibility. There should be an additional indicator for status besides color on the map view.
